app/src/main/java/uc/edu/klopfsea/groupscheduler/model/Event.kt

```diff
@@ -12,8 +12,8 @@ data class Event(
 ) {
     // property declared in class body
     var id = 0
-    var month = 0
-    var year = 0
+    private var month = 0
+    private var year = 0
```

Comment:
these variables could be set to private, and therefore should be after any public variables according to these kotlin conventions, https://kotlinlang.org/docs/coding-conventions.html#modifiers-order